examples/dedicated_vmseries_and_autoscale/example.tfvars

```diff
@@ -0,0 +1,281 @@
+# --- GENERAL --- #
+location            = "North Europe"
+resource_group_name = "autoscale-common"
+name_prefix         = "example-"
+tags = {
+  "CreatedBy"   = "Palo Alto Networks"
+  "CreatedWith" = "Terraform"
+}
+enable_zones = false
+
+# --- VNET PART --- #
+vnets = {
+  "transit" = {
+    name          = "transit"
+    address_space = ["10.0.0.0/25"]
+    network_security_groups = {
+      "management" = {
+        name = "mgmt-nsg"
+        rules = {
+          vmseries_mgmt_allow_inbound = {
+            priority                   = 100
+            direction                  = "Inbound"
+            access                     = "Allow"
+            protocol                   = "Tcp"
+            source_address_prefixes    = ["134.238.135.137", "130.41.247.15"]
+            source_port_range          = "*"
+            destination_address_prefix = "10.0.0.0/28"
+            destination_port_ranges    = ["22", "443"]
+          }
+        }
+      }
+      "public" = {
+        name = "public-nsg"
+      }
```

Comment:
nope, you do not define NSGs for AppGWs, you can if you really want to limit traffic, but normally an APPGW subnet is publicly open

Reply:
think of it from this perspective - you put an AppGW in front of an HTTP application (webpage, REST endpoint, etc). An HTTP app is normally available to anyone. If you want to hide it or you want to limit access to it, you do it internal. Of course you can come up with a scenario where IP filtering for a web page could be handy, but I'm guessing:
1. we want to make this example simple
2. there are couple of other/better methods to achieve it than limiting access by IPv4

Reply:
Ok, I understand your arguments. Of course my suggestion can be skipped. 

I was only thinking in the same way we have currently for frontend LB, which has public IP and which also have NSG assigned to protect traffic using rules based on IP addresses and ports:

```
load_balancers = {
  "public" = {
    name                        = "public-lb"
    network_security_group_name = "example-public-nsg"
    network_security_allow_source_ips = [
      #  "x.x.x.x", # Put your own public IP address here  <-- TODO to be adjusted by the customer
      "0.0.0.0/0",
    ]
...
```

Reply:
IMO we should skip that part as well, looks a little bit strange to put a *firewall* in front of a **Firewall**.

On the other hand, NSG is different, this is TCP/UDP traffic, this can be i.e. some Site2Site VPN setting that we want to open to specific IPs only. But for HTTP traffic I would say have it public

Reply:
ok, let's skip that suggestion and do as you propose 👍